<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTaskSettingsForm</class>
 <widget class="QDialog" name="TTaskSettingsForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>%1 - Task Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/TaskSettings_16x16</normaloff>:/TaskSettings_16x16</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="Tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="TotalCalc">
         <property name="text">
          <string>Calculate count and total size of files</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckFreeSpace">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Check free space</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="CopyDateTime">
         <property name="text">
          <string>Copy date and time of files and folders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CopyAttr">
         <property name="text">
          <string>Copy attributes of files and folders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CopyHidden">
         <property name="text">
          <string>Copy hidden files and folders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CopySystem">
         <property name="text">
          <string>Copy system files and folders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="FollowShortcuts">
         <property name="text">
          <string>Follow shortcuts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CopyEmptyDirs">
         <property name="text">
          <string>Copy empty folders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NoCreateRootDir">
         <property name="text">
          <string>Don't create root folders</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="SubDirsDepth_Label">
           <property name="text">
            <string>Depth of copying of subdirectories:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SubDirsDepth">
           <property name="specialValueText">
            <string>Unlim.</string>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RAMTab">
      <attribute name="title">
       <string>Memory</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="WarningIcon">
           <property name="text">
            <string notr="true">Warning
Icon</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="WarningLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;&lt;font color=&quot;red&quot;&gt;Don't change, if are not assured!&lt;/font&gt;&lt;/b&gt;&lt;/p&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="RAMBuffer">
         <property name="title">
          <string>RAM Buffer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="AutodetectRAM">
            <property name="text">
             <string>Autodetect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="RAMBox" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="CellSize_Label">
               <property name="text">
                <string>Cell size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="CellSize">
               <property name="suffix">
                <string> MB</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CellsCount_Label">
               <property name="text">
                <string>Cells count:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="CellsCount">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="RequiredRAM_Label">
               <property name="text">
                <string>Required RAM:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="RequiredRAM">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="LockMemory">
            <property name="toolTip">
             <string>&lt;p&gt;Memory lock whenever possible forbids writes of memory pages in the swap file.&lt;/p&gt;
&lt;p&gt;&lt;font color=&quot;red&quot;&gt;Locking memory pages may degrade the performance of the system and another applications!&lt;/font&gt;&lt;/p&gt;</string>
            </property>
            <property name="text">
             <string>Lock Memory (if available)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NoUseCache">
         <property name="text">
          <string>Don't use system cache</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>For files larger than</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="NoUseCacheForNum">
            <property name="maximum">
             <number>1023</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="NoUseCacheForUnit">
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GB</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Default">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TotalCalc</tabstop>
  <tabstop>SubDirsDepth</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>TTaskSettingsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CellsCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TTaskSettingsForm</receiver>
   <slot>calculateRAM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AutodetectRAM</sender>
   <signal>toggled(bool)</signal>
   <receiver>RAMBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CellSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TTaskSettingsForm</receiver>
   <slot>calculateRAM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TotalCalc</sender>
   <signal>toggled(bool)</signal>
   <receiver>CheckFreeSpace</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NoUseCache</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculateRAM()</slot>
 </slots>
</ui>
